examples/pytorch/mnist_trainer.py

```diff
@@ -23,11 +24,11 @@
 if module_available("lightning"):
     from lightning.pytorch import Trainer, seed_everything
     from lightning.pytorch.demos.mnist_datamodule import MNISTDataModule
-    from lightning.pytorch.plugins.precision import MixedPrecisionPlugin
+    from lightning.pytorch.plugins.precision import MixedPrecision
```

Comment:
Shall we then also bump min dependency?

Reply:
Is it not implicit based on the support matrix per release ?